examples/client/http/with_mtls.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 protocol=https
 host=localhost
-port=8088
+port=8008
 path="messages"
```

Comment:
Please change value to `8088`, because http mtls server in `cmd/main.go` run at 8088